pom.xml

```diff
@@ -90,9 +90,9 @@
         <accumulo.version>1.8.1</accumulo.version>
         <zookeeper.version>3.4.10</zookeeper.version>
         <thrift.version>0.9.3</thrift.version>
-        <hadoop.version>2.6.4</hadoop.version>
+        <hadoop.version>2.8.4</hadoop.version>
         <spark.version>2.2.0</spark.version>
-        <hbase.version>1.3.1</hbase.version>
+        <hbase.version>1.4.4</hbase.version>
```

Comment:
oops, did you mean to modify this file? I don't think we want to change versions yet

Reply:
Arg, keep having to add that to test...